src/iri/utils.ts

```diff
@@ -20,12 +21,15 @@ export function isIRI( iri:string ):boolean {
 	return hasProtocol( iri ) || ! isAbsolute( iri );
 }
 
+export function isBNodeLabel( label:string ):boolean {
+	return /^_:/.test( label );
```

Comment:
Instead of using a new RegExp object each time the function is executed, pull the object to a constant outside of the function to reuse it